test/packaging/ansible/canaries.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I would remove this file as canaries are being added in https://github.com/newrelic/opentelemetry-collector-releases/pull/14

Reply:
👍  removed in e93192faeaa6c9c1ba6df217674f6012818fb4a3

Reply:
👍